catalog-info.yaml

```
@@ -0,0 +1,28 @@
+---
+ # yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+ apiVersion: backstage.io/v1alpha1
+ kind: Resource
+ metadata:
+   name: buildkite-pipeline-package-registry
+   description: Buildkite Pipeline for package-registry
+   links:
+     - title: Pipeline
+       url: https://buildkite.com/elastic/package-registry
+
+ spec:
+   type: buildkite-pipeline
+   owner: group:ingest-fp
+   system: buildkite
+   implementation:
+     apiVersion: buildkite.elastic.dev/v1
+     kind: Pipeline
+     metadata:
+       name: package-registry
```

Add the description here 

```suggestion
       name: package-registry
       description: Buildkite Pipeline for the Package Registry project
```